core/txpool/legacypool/legacypool.go

```diff
@@ -615,9 +544,6 @@ func (pool *LegacyPool) validateTxBasics(tx *types.Transaction, local bool) erro
 		MaxSize: txMaxSize,
 		MinTip:  pool.gasTip.Load().ToBig(),
 	}
-	if local {
-		opts.MinTip = new(big.Int)
```

Comment:
Do I understand correctly that txs from prioritized addresses are now also subject to the minimum tip threshold and there is no way to force accept certain txs with low tip? If so then are you sure this is the right behavior? I would think that we should still set `MinTip` to zero if it's signed by one of the prio address set since the user's intention there was to include those txs whenever possible regardless of low price.

Reply:
The transaction pool does not have a concept of local transactions (or prioritized addresses). 
The mintip enforcement or exception is instead a concern for miner, during block building

Reply:
But to answer the question. See worker.go
```golang
	miner.confMu.RLock()
	tip := miner.config.GasPrice
	prio := miner.prio
	miner.confMu.RUnlock()

	// Retrieve the pending transactions pre-filtered by the 1559/4844 dynamic fees
	filter := txpool.PendingFilter{
		MinTip: uint256.MustFromBig(tip),
	}
```
So yes, the filter which we use to pick them from the pool does use the `mintip` indiscriminately, i.e the `miner.GasPrice`. _After_ the mintip-filtering, the `prio` set is included first.

Reply:
I think I understand the concern from Zsolt.

Originally, the local transactions can bypass the minTip restrictions. Now, they will always be rejected by txpool, regardless of it's received from local RPC or p2p network.

With the local tracker, the local transactions will be persisted and be resubmitted all the time, but none of the attempt will succeed

Reply:
Yes, I meant what Gary has just described, so the behavior has changed and we consider it acceptable. I have no concerns with the low tip txs not being propagated over network, I had some slight concerns about not being able to force feed them to the miner anyways, but I will not insist on this being a serious problem :) I like the concept that the pool does not discriminate. One option to solve this would be if the miner could also somehow be connected to the locals tracker and get the local/prioritized txs that cannot get into the tx pool at the moment. But I do not insist that we should do this and the PR is good as it is.